src/ember-app.js

```diff
@@ -136,8 +136,12 @@ class EmberApp {
       let isWhitelisted = whitelist.indexOf(packageName) > -1;
 
       if (isWhitelisted) {
-        let resolvedModulePath = resolve.sync(moduleName, { basedir: distPath });
-        return require(resolvedModulePath);
+        try {
+          let resolvedModulePath = resolve.sync(moduleName, { basedir: distPath });
+          return require(resolvedModulePath);
+        } catch (error) {
+          return require(moduleName);
```

Comment:
Is it fair to say this is _only_ applicable to the `development` case when `distPath` has no `node_modules`?

Reply:
Yes this is what this PR proposes. I asked the question about deployed case at https://github.com/ember-fastboot/fastboot/issues/198 and dist/node_modules is requried.

I updated the readme about deployment instructions.

Reply:
http://ember-fastboot.com/docs/user-guide#building-for-production should also be updated if PR is going to merge.